ui/src/components/sidemenu/siteMenu.tsx

```diff
@@ -4,13 +4,19 @@ import {
   DatabaseOutlined,
   EyeOutlined,
   RocketOutlined,
+  ControlOutlined,
 } from "@ant-design/icons";
 import { Link } from "react-router-dom";
 
 const { Title } = Typography;
 const { Sider } = Layout;
 
 const SideMenu = () => {
+  const enableRBAC = window.environment?.enableRBAC;
```

Comment:
>enableRBAC [](http://example.com/codeflow?start=8&length=10)

let's add a type on enableRBAC & showManagement, this will make code more rebust